python/glide-shared/glide_shared/config.py

```
@@ -1203,11 +1230,15 @@ def __init__(
         refresh_topology_from_initial_nodes: bool = False,
         tcp_nodelay: Optional[bool] = None,
         pubsub_reconciliation_interval: Optional[int] = None,
+        circuit_breaker: Optional[
+            "CircuitBreakerConfiguration"
+        ] = CircuitBreakerConfiguration(),
```

Ruff (0.15.12)

[warning] Do not perform function call `CircuitBreakerConfiguration` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)